src/aiida/tools/dumping/utils.py

```diff
@@ -14,29 +14,35 @@
 import shutil
 from pathlib import Path
 
-__all__ = ['validate_make_dump_path']
+__all__ = ['_prepare_dump_path']
```

Comment:
If you don't want this function to be used outside of the module don't expose it.

Reply:
On second thought, I think it's actually OK that the function is being used outside. So I'll mark it not private and expose it 👍🏽